python/meterpreter/meterpreter.py

```diff
@@ -22,14 +22,6 @@
 else:
 	has_windll = hasattr(ctypes, 'windll')
 
-# this MUST be imported for urllib to work on OSX
```

Comment:
I remember the two `isinstance` blocks were to deal with a difference in how versions of Python 3 have `__builtins__` defined. As for the rest of the changes I don't quite remember.